### ISSUE TITLE
Roads with bicycle access are rendered with red cross

### ISSUE THREAD

**Issue by Florimondable** [mattesCZ/mtbmap#118]
This road https://www.openstreetmap.org/way/36889817/history
is rendered with red cross though no tags saying it is private or no bicycle access on it.
http://mtbmap.cz/#zoom=12&lat=49.705&lon=4.0261

**Comment by mattesCZ** [comment]
Thanks for noticing. It looks like it's derived from some public transport relation https://www.openstreetmap.org/relation/5574491
Can you please check, if it's mapped correctly? If it is, I will try to fix the map style.

**Comment by Florimondable** [comment]
Yes it's well tagged: it's a bus route relation where it's forbidden to carry your bicycle.
It's documented into the wiki here https://wiki.openstreetmap.org/wiki/Tag:route%3Dbus
